src/components/Like/LikeBtn.module.scss

```diff
@@ -23,11 +23,13 @@
 }
 
 .small {
-  height: 12px;
   stroke-width: 0;
-  margin-top: -2px;
   fill: var(--casper);
```

Comment:
The only necessary change for fixing the bug was to move `fill: var(--casper);` inside the `grey` class

Reply:
yes, it can be true, if we didn't have "like" without fill. So, I'm waiting Andrey for new desing for likes, he said, that likes should be another